CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## [Unreleased]
 ### Added
 - active/Cross Site WebSocket Hijacking.js > an active scan for Cross-Site WebSocket Hijacking vulnerability
+- targeted/cve-2021-22214.js > A targeted script to check for Unauthorised SSRF on Gitlab - CVE 2021-22214
```

Comment:
`Gitlab` → `GitLab`, same comment for other occurrences (also pull request title).

Reply:
Updated.

Reply:
This one was not updated.